File: joseffe-barroso/Python/exercise14.py
heigth = float(input("Digite a altura: "))
catetoOne = float(input("Digite o primeiro cateto: "))
catetoTwo = float(input("Digite o primeiro cateto: "))

catetoSum = catetoOne**2 + catetoTwo**2

if (heigth**2 == catetoSum):
    print("É um triângulo retangulo")

else:
    print("Não é um triângulo retangulo")

File: joseffe-barroso/Java/src/entrada_processamento_saida/Exercise14.java
package entrada_processamento_saida;

import java.util.Scanner;

public class Exercise14 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		
		double heigth, catetoOne, catetoTwo, catetoSum;
		
		System.out.print("Digite a altura: ");
		heigth = scanner.nextDouble();
		
		System.out.print("Digite o primeiro cateto: ");
		catetoOne = scanner.nextDouble();
		
		System.out.print("Digite o primeiro cateto: ");
		catetoTwo = scanner.nextDouble();
		
		catetoSum = (catetoOne * catetoOne) + (catetoTwo * catetoTwo);
		
		if ((heigth * heigth) == catetoSum) {
			System.out.println("É um triângulo retângulo");
		} else {
			System.out.println("Não é um triângulo retângulo");
		}

		scanner.close();
	}
}
